code/game/gamemodes/events/holidays/Holidays.dm

```diff
@@ -188,6 +190,9 @@ var/global/Holiday = null
 		to_chat(world, "<h4>Happy [Holiday] Everybody!</h4>")
 		if(Holiday == XMAS_EVE || Holiday == XMAS)
 			Christmas_Game_Start()
+		if(Holiday == INTERNATIONAL_CLOWN_DAY)
+			sleep(60)
```

Comment:
This just makes code rainbow after 6 seconds, not 60. For info.

Reply:
Understood